backend/src/v5/processors/teamspaces/projects/models/commons/tickets.js

```diff
@@ -23,6 +23,8 @@ const {
 	modulePropertyLabels,
```

Comment:
Let's say I have 2 users in the same project:
- The one on the left is looking at a given federation item from the dashboard;
- The one on the right is looking at the same federation in the viewer;
![image](https://github.com/user-attachments/assets/3dfc984a-a775-430e-954f-bc7ba1e59b78)

The user on the right closes/re-opens a ticket
Should the user on the left see the tickets count updating in realtime?

Reply:
I think this could be a separate issue.

Reply:
No from what I see only FEDERATION_UPDATE_TICKET event is thrown when a ticket is updated :/

Reply:
moved this into https://github.com/3drepo/3D-Repo-Product-Team/issues/557